src/args.rs

```diff
@@ -397,4 +398,12 @@ pub(crate) fn clap_app(tmp_dir: &str) -> clap::App {
                 .env("UNFTP_PROXY_EXTERNAL_CONTROL_PORT")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name(ENABLE_SITEMD5)
+                .long("enable-sitemd5")
+                .value_name("TIMEOUT_SECONDS")
```

Comment:
Drop the value_name assignment since there is no value

Reply:
indeed.....